PROTOCOL.md

```diff
@@ -1057,27 +1059,48 @@ When Row Tracking is enabled (when the table property `delta.enableRowTracking`
 
 The Clustered Table feature facilitates the physical clustering of rows that share similar values on a predefined set of clustering columns.
 This enhances query performance when selective filters are applied to these clustering columns through data skipping.
-Clustering columns must be specified during the initial definition of a clustered table, and they can be modified after the table has been created.
+Clustering columns can be sprecified when creating a table or later, as long as the table doesn't have partition columns.
```

Comment:
FYI @imback82 / @ryan-johnson-databricks , I have updated here and [below](https://github.com/delta-io/delta/pull/2294/files#diff-62d56fa1c25dda8d49ccdbf9ea31df4576f84625706d54613dff0acc8f974d97R1070) to let clustering table feature to be enabled either during table creation or at a later stage